src/control/lib/ipmctl/nvm_test.go

```diff
@@ -29,8 +29,9 @@ func skipNoPerms(t *testing.T) {
 	if u.Uid != "0" {
 		// Alert the user even if they're not running the tests in verbose mode
 		fmt.Printf("%s must be run as root\n", t.Name())
-		t.Skip("test doesn't have NVM API permissions")
+		return true
```

Comment:
Could be useful to preserve the log message as you have done on _syslog_test.go_.
I will also explicitly add the information that the test is skipped.
IF you think that this last remark is relevant, such information could also be added in skipped test from _syslog_test.go_.
```suggestion
                t.Log("SKIP TEST: test doesn't have NVM API permissions")
		return true
```